src/main/java/org/fundacionjala/coding/escarleth/movies/MovieNewRelease.java

```diff
@@ -0,0 +1,33 @@
+package org.fundacionjala.coding.escarleth.movies;
+
+/**
+ * Powered by Escarleth L.
+ */
+public class MovieNewRelease extends Movie {
+    private static final int AMOUNT = 3;
+
+    /**
+     * Constructor for MovieNewRelease.
+     *
+     * @param title of the Movie.
+     */
+    public MovieNewRelease(String title) {
+        super(title);
+    }
+
+    /** determineRentalAmount method.
+     * {@inheritDoc}
+     */
+    @Override
+    double determineRentalAmount(int daysRented) {
+        return daysRented * AMOUNT;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Cast one of the operands of this multiplication operation to a "double". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2184)